ape_alchemy/provider.py

```diff
@@ -106,7 +104,12 @@ def uri(self):
                 break
 
         if not key:
-            raise MissingProjectKeyError(options)
+            env_var_str = ", ".join([f"${n}" for n in options])
+            error_message = f"Using demo key. Set one of {env_var_str}."
+            logger.warning(error_message)
+
+            # Alchemy allows you to use the "demo" key for simple, demo purposes.
+            key = "demo"
```

Comment:
Does this actually function?

Reply:
It does! It is super rate limited though